demux/demux_playlist.c

```diff
@@ -21,6 +21,8 @@
 #include <dirent.h>
 
 #include <libavutil/common.h>
+#include <libxml/parser.h>
```

Comment:
#if HAVE_LIBXML2

Reply:
Good point. I've added a few `#if`'s and it works as expected now with `--disable-libxml2`.